### DISCUSSION TITLE
"A Playstation 4 emulator just begin" is not correct English

### DISCUSSION THREAD

**Discussion by VelocityRa** [Inori/GPCS4#112] [General]
The project description makes no syntactical or grammatical sense.
A correct way to say this would be "A work-in-progress PlayStation 4 emulator." or "Researching PlayStation 4 emulation." or something similar.

**Comment by Inori** [comment]
FIxed.
I remember there is [another guy kicked me](https://github.com/Inori/GPCS4/pull/46) for my poor English.

**Reply by VelocityRa** [reply]
Haha don't worry it's not too bad.
